pkg/console/subresource/consoleserver/config_builder.go

```diff
@@ -74,6 +75,11 @@ func (b *ConsoleServerCLIConfigBuilder) StatusPageID(id string) *ConsoleServerCL
 	return b
 }
 
+func (b *ConsoleServerCLIConfigBuilder) ProxyTrustedCAFile() *ConsoleServerCLIConfigBuilder {
```

Comment:
Wasnt really sure if we wanna red rid of this, since the dir should be and api. But probably cant hurt, since we are setting the  `oauthEndpointCAFile: /var/run/secrets/kubernetes.io/serviceaccount/ca.crt` as well.